python/tox.ini

```diff
@@ -1,17 +1,25 @@
 [tox]
-envlist = py{36,37,38,39}
+envlist = py{37,38,39,310,311}
 skipsdist = True
-skip_missing_interpreters = True
+skip_missing_interpreters = False
 ignore_basepython_conflict = True
 
 [testenv]
 basepython = python3
 sitepackages = True
 setenv =
     PYTHONPATH = {toxinidir}:{toxinidir}
+deps =
+    pip
+    setuptools
+    wheel
+    aiohttp
+    cryptography
+    requests
 commands =
-    python setup.py develop
-    python test/test.py
+    pip install -e .
+    python ccxt/test/test_async.py gateio
```

Comment:
@carlosmiei, I couldn't find the file test/test.py so I put test_async and test_sync instead, but not sure what exchange or arguments we want to pass in, or do we want to do something similar to run-tests.js ?

Reply:
@pcriadoperez If we just want to run something very quickly, I would suggest running the brokerId tests `python ccxt/test/test_async.py  --idTests`, that should be enough. If you call it normally it will run live tests that perform API requests and take much longer.